### DISCUSSION TITLE
Kaggle AutoGluon Multimodal installation (April 2024)

### DISCUSSION THREAD

**Discussion by samvelkoch** [autogluon/autogluon#4043] [Q&A]
It seems I stucked to setup AutoGluon Multimodal v.1 at Kaggle these days. Any working CPU and GPU examples ? I've tried all known to me possible tricks to make AG work at Kaggle but no success for Mulitmodal (meanwhile Tabular and TS works fine for me)

**Comment by hohoCode** [comment]
A very important issue.
